CDTDatastoreReplicationAcceptanceTests/ReplicationAcceptance.m

```diff
@@ -1492,7 +1491,13 @@ -(void)testCreateReplicationWithExtraHeaders
     NSDictionary *extraHeaders = @{@"SpecialHeader": @"foo", @"user-agent":userAgent};
     pull.optionalHeaders = extraHeaders;
     
-    NSDictionary *pullDoc = [pull dictionaryForReplicatorDocument:nil];
+    NSMutableDictionary *pullDoc = [[NSMutableDictionary alloc] init];
```

Comment:
You can rewrite this test without reference to `pullDoc`, as it's no longer needed. Likewise you can remove the asserts below which reference `pullDoc`.